sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/bitwiseExpressions.scala

```diff
@@ -204,3 +205,47 @@ case class BitwiseCount(child: Expression)
     case LongType => java.lang.Long.bitCount(input.asInstanceOf[Long])
   }
 }
+
+object BitwiseGetUtil {
+  def calculate(target: Long, pos: Int): Long = {
+    if (pos < 0) {
+      throw new IllegalArgumentException(s"Invalid bit position: $pos less than zero")
+    } else if (java.lang.Long.SIZE <= pos) {
+      throw new IllegalArgumentException(s"Invalid bit position: $pos exceeds the " +
+        "bit length of the target long")
+    }
+    (target >> pos) & 1
+  }
+}
+
+@ExpressionDescription(
+  usage = "_FUNC_(expr, pos) - Returns the value of the bit (0 or 1) at the specified position.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_(11, 0);
+       1
+      > SELECT _FUNC_(11, 2);
+       0
+  """,
+  since = "3.2.0",
+  group = "bitwise_funcs")
+case class BitwiseGet(left: Expression, right: Expression)
+  extends BinaryExpression with ImplicitCastInputTypes with NullIntolerant {
+
+  override def inputTypes: Seq[IntegralType] = Seq(LongType, IntegerType)
+
+  override def dataType: DataType = LongType
```

Comment:
if it only returns 0 or 1, why we use long type? how about other databases?

Reply:
Return type: Same as the input value

Reply:
We decided use byte type.